drivers/hmis/spec/requests/hmis/update_inventory_spec.rb

```diff
@@ -120,6 +122,58 @@
         expect(errors[0]['type']).to eq 'required'
       end
     end
+
+    it 'validates start date against project operating period' do
```

Comment:
I think we should check that it passes validation if the inventory start/end are on the project start end.    I'm having trouble telling if that check is here.

Reply:
Thanks - I added a bunch more tests